# HG changeset patch
# User John Dennis <jdennis@redhat.com>
# Date 1420558515 18000
#      Tue Jan 06 10:35:15 2015 -0500
# Node ID 22895012dc87281141bf15494bb860e68b37ecc6
# Parent  46403a1eb97050a09eb8c4e6adf428c742ab9c5f
fix install_doc manifest, it omitted the run_tests script.

Tighten up the manifest regular expressions.
Exclude the pki directory from doc/examples and test.

diff --git a/setup.py b/setup.py
--- a/setup.py
+++ b/setup.py
@@ -21,9 +21,11 @@
 
 doc_manifest = [
     [['include README LICENSE* doc/ChangeLog',
-      'recursive-include doc *.py *.txt',],
+      'recursive-include doc *.py *.txt',
+      'prune doc/examples/pki'],
      [('^doc/', '')], None],
-    [['recursive-include test *.py *.txt',],
+    [['recursive-include test run_tests setup_certs.py test_*.py util.py *.txt',
+      'prune test/pki'],
      None , None],
     [['recursive-include lib *.py *.txt',],
      [('^lib/', '')] , 'examples'],
